### PR TITLE
[EC-140] Avoid duplication in configuration files

### PR DESCRIPTION
**Pull request by AlanVerbner**
## Description

In order to avoid duplicating our config in universal config and src/main/resources/application.conf...

1. Rename src/main/resources/application.conf  to src/main/resources/resources.conf 
2. Remove all config from the ./src/universal/conf/application.conf
3. Put in the line include "resources.conf"  in ./src/universal/conf/application.conf
4. Put in an example override of the kind etc-client.network.server-address.port - 90

### Notes
Please test overwriting a config, performing `sbt dist` and running the app

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -1,104 +1 @@
...
-
-}
-
-akka {
-  loggers = ["akka.event.slf4j.Slf4jLogger"]
-  loglevel = "DEBUG"
-  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
-  logger-startup-timeout = 30s
-}
+include "resources.conf"
```
Comment (mcsherrylabs) [inline src/main/resources/application.conf]: Presuming this was necessary to run it from sbt?

Comment (AlanVerbner) [inline src/main/resources/application.conf]: @mcsherrylabs yes...without it I cannot `sbt run`

Comment (mcsherrylabs) [inline src/main/resources/application.conf]: Actually I think the correct name for that file might be 'reference.conf'... not resources.conf.

**Comment by AlanVerbner** [comment]
@rtkaczyk i've addressed your comment including #98 changes in this PR

**Pull request merged**